### PR TITLE
Update iOS SDK v2.2.0 release notes

### PR DESCRIPTION
**Pull request by vonageCI**
iOS SDK v2.2.0 notes added to `release-notes.md`.

### CODE REVIEW

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,17 @@ navigation_weight: 0
 
 # Release Notes
 
+## 2.2.0 - 2020-04-22
```
Comment (tbedford) [inline _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md:9]: Headings/subheadings should always have a new line after them as per Markdown spec and previous entries in this changlog.

**Comment by mheap** [comment]
@pardel Who can help progress this?

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md [hunk 2]
```diff
@@ -6,6 +6,18 @@ navigation_weight: 0
...
+
+### Added
```
Comment (tbedford) [inline _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md:11]: New line after subheading required.

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md [hunk 3]
```diff
@@ -6,6 +6,18 @@ navigation_weight: 0
...
+- Added `isConnected` method to `NXMClient` to show current connection state.
+
+```objective_c
+[NXNClient.shared isConnected]
+```
+
+### Fixed
```
Comment (tbedford) [inline _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md:18]: New line after subheading required.

**Pull request merged**